src/lib/handlers/db.js

```diff
@@ -42,26 +47,37 @@ const createUserDocument = async (user) => {
 
 export const deleteTrick = async (trickId) => {
 	try {
-		const isConfirm = confirm('Are you sure?')
-		if(!isConfirm) return
-		setLoading(true)
+		const isConfirm = confirm('Are you sure?');
+		if (!isConfirm) return;
+		setLoading(true);
 		const userRef = doc(db, 'users', user.uid);
```

Comment:
I would make these (`'users', 'tricks'`) constants, in a separate `constants` file, so if it ever changes it's more constistent